contracts/contracts/token/WrappedOusd.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { ERC20 } from "../../lib/solmate/src/tokens/ERC20.sol";
+import { ERC4626 } from "../../lib/solmate/src/mixins/ERC4626.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { OUSD } from "./OUSD.sol";
+import { Governable } from "../governance/Governable.sol";
+
+contract WrappedOusd is ERC4626, Governable {
+    using SafeERC20 for IERC20;
+
+    constructor(
+        ERC20 _underlying,
+        string memory _name,
+        string memory _symbol
+    ) ERC4626(_underlying, _name, _symbol) Governable() {
+        OUSD(address(_underlying)).rebaseOptOut(); // It's not treated as a contract yet
```

Comment:
nit: add a comment to explain that calling rebaseOptOut(), followed by rebaseOptIn() allows to opt-in for rebasing in the constructor vs having to call rebaseOptIn() outside of the constructor. nice trick :)

Reply:
With the change to using a proxy, we no longer use this silly technique, and now just rebaseOptIn() in the initializer.